spki

```diff
@@ -318,6 +322,21 @@ verify () {
 	|| { echoc 'Could not verify certificate chain of trust' red 1>&2;  return 1; }
 }
 
+truststore () {
+	CERT=$INTRMDT_DIR/certs/ca-chain.cert.pem
+	TRUSTSTORE=$INTRMDT_DIR/certs/"$1"_truststore.p12
+	if [[ ! -r  $CERT ]]; then
+		echoc "Could not find '$1'" red 1>&2
+		return 1
+	else
+	  INTRMDT_PASS=$(get-password "truststore for $1" -noverify)
```

Comment:
Let's add a check here that `keytool` actually exists on the system (eg `which keytool`). OpenSSL is an obvious dependency for the script, but not everyone might have `keytool`.

We should also add a note in the documentation